web/src/components/molecules/Schema/FieldModal/FieldUpdateModal/index.tsx

```diff
@@ -47,6 +48,7 @@ const FieldUpdateModal: React.FC<Props> = ({
 }) => {
   const [form] = Form.useForm();
   const { TabPane } = Tabs;
+  const selectValues = Form.useWatch("values", form);
```

Comment:
Does `currentValues` make more sense here? If `select` makes more sense to you I think this needs to be `selectedValues`

Reply:
@KaWaite , yup you are right.
`selectedValues` will be, at it is related to select field only in this file